course_discovery/apps/api/v1/tests/test_views/test_courses.py

```diff
@@ -603,6 +603,7 @@ def create_course_and_course_run(self, data=None, update=True):
             body=json.dumps({'access_token': 'abcd', 'expires_in': 60}),
             status=200,
         )
+        # pylint: disable=useless-suppression
```

Comment:
Why this and the other below? I don't see further pylint suppressions in this code block.

Reply:
I had problems with pylint and it was a workaround. Perhaps pylint version increased and similar tricks began to appear. I'll double-check.

Reply:
So `# pylint: disable=useless-suppression` there is no such disabler anymore. I deleted them in my next commits and you had a deal with the old implementation I guess.